diff -r ca4999c5c60e setup.py
--- a/setup.py	Tue May 24 13:00:18 2011 +0200
+++ b/setup.py	Wed May 25 11:42:31 2011 +0200
@@ -38,7 +38,7 @@
             'lighttpd_root_fix = mapproxy.util.wsgi:lighttpd_root_fix_filter_factory',
         ],
     },
-    package_data = {'': ['*.xml', '*.yaml', '*.ttf', '*.wsgi', '*.ini']},
+    package_data = {'': ['*.xml', '*.yaml', '*.wsgi', '*.ini']},
     install_requires=install_requires,
     classifiers=[
         "Development Status :: 5 - Production/Stable",
