Index: mapproxy/setup.py
===================================================================
--- mapproxy.orig/setup.py	2014-09-03 12:22:34.929326173 +0200
+++ mapproxy/setup.py	2014-09-03 13:49:14.474139720 +0200
@@ -80,7 +80,7 @@
             'lighttpd_root_fix = mapproxy.util.wsgi:lighttpd_root_fix_filter_factory',
         ],
     },
-    package_data = {'': ['*.xml', '*.yaml', '*.ttf', '*.wsgi', '*.ini']},
+    package_data = {'': ['*.xml', '*.yaml', '*.wsgi', '*.ini']},
     install_requires=install_requires,
     classifiers=[
         "Development Status :: 5 - Production/Stable",
